README.md

```diff
@@ -54,6 +54,8 @@ Testing
 #### Run example copydb usage
 
 - `make copydb && ghostferry-copydb -verbose examples/copydb/conf.json`
+- In your browser you can visit http://localhost:8000/ to access the UI
+- Click on `Allow Automatic Cutover` button
```

Comment:
Isn't this in the tutorial already?

Reply:
It definitely is in the tutorial. I thought it would be helpful to mention here at the time. I can remove it.